tests/data/bintec-beip-plus.json

```diff
@@ -37,8 +37,6 @@
                     "ifSpeed_prev": null,
                     "ifConnectorPresent": null,
                     "ifPromiscuousMode": null,
-                    "ifHighSpeed": null,
```

Comment:
For some reason, I had to remove all lines including `ifHighSpeed*` from test data, else the tests would fail (it would expect these lines not to be in the test file). Even after re-creating the test and snmprec files from scratch, these props would still be there and lead to failing tests. I searched for changes regarding these props in upstream, but was not yet able to understand the changes. If removing these props from test data is an issue, please let me know how to deal with this.